test/e2e/airgap.go

```diff
@@ -0,0 +1,20 @@
+package e2e
+
+import (
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+// Run airgap deployment but allow bootstrap cluster to access local peers.
```

Comment:
small nit: could we follow the convention we use elsewhere and start with the function name? `runVSphereAirgapConfigFlow`?